analysis/analysis_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"testing"
+)
+
+// TestProcessOpenAPIDocs is a simple test that tests that filehander
+// is able to correctly process the OpenAPIdocuments in a given
+// directory
+func TestProcessOpenAPIDocs(t *testing.T) {
```

Comment:
nit: it's common practice to name a test after the method it's testing.

Reply:
now named TestGenerateAnalysis